### PR TITLE
Update Parlot

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by sebastienros** [comment]
@MikeAlhayek

**Comment by sebastienros** [comment]
This is using Parlot 1.0.2 but the other libraries should not need to be updated as there are no breaking changes. Pinning OC to 1.0.2 would work too.

File: src/Shortcodes/ShortcodesParser.cs [hunk 1]
```diff
@@ -128,16 +128,17 @@ private Shortcode ParseShortcode()
                 {
                     arguments ??= CreateArgumentsDictionary();
 
-                    arguments[argumentIndex.ToString()] = Character.DecodeString(new TextSpan(_scanner.Buffer, _result.Start + 1, _result.Length - 2)).ToString();
+                    arguments[argumentIndex.ToString()] = Character.DecodeString(_result)[1..^1].ToString();
```
Comment (MikeAlhayek) [inline src/Shortcodes/ShortcodesParser.cs:131]: Got yo lobe this syntax :)

File: tests/Shortcodes.Tests/Shortcodes.Tests.csproj [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
     <DebugSymbols>true</DebugSymbols>
     <DebugType>pdbonly</DebugType>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-    <PackageReference Include="coverlet.collector" Version="1.2.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.0-release-24352-06" />
```
Comment (lahma) [inline tests/Shortcodes.Tests/Shortcodes.Tests.csproj:12]: Are you sure about this version?

Comment (sebastienros) [reply tests/Shortcodes.Tests/Shortcodes.Tests.csproj:12]: NuGet package manager did it by itself, had not checked

Comment (sebastienros) [reply tests/Shortcodes.Tests/Shortcodes.Tests.csproj]: I had the pre-release checkbox

**Pull request merged**